packages/block-library/src/post-featured-image/index.php

```diff
@@ -14,11 +14,6 @@
  * @return string Returns the featured image for the current post.
  */
 function render_block_core_post_featured_image( $attributes, $content, $block ) {
-	if ( ! isset( $block->context['postId'] ) ) {
-		return '';
-	}
-	$post_ID = $block->context['postId'];
```

Comment:
Now that this isn't used here anymore, should we also remove this from being passed as context? I'm not very familiar with the internals of that part, so could use some guidance around that.

Reply:
I think we should leave the `isset( $block->context['postId']` guard clause. It makes sure that blocks are rendered in the loop context or closes check we've got to that :)

Reply:
Makes sense, added back the checks.